### PR TITLE
Formatting the user page with basic sections, on going

### PR DESCRIPTION
**Pull request by linkaren**

### CODE REVIEW

**Comment by FluffySheep** [comment]
Picture please!

**Comment by linkaren** [comment]
Screen Shot 2019-03-24 at 5 42 30 PM

File: src/main/webapp/user-page.html [hunk 1]
```diff
@@ -55,37 +137,40 @@ <h1 id="page-title">User Page</h1>
...
+                }
+              } )
+              .then( editor => { console.log( editor ); })
+              .catch( error => { console.log( error ); });
+          </script> 
+        </form>
+      </article>
+
+      <hr/>
+
```
Comment (FluffySheep) [inline src/main/webapp/user-page.html]: nit: remove extra space

File: src/main/webapp/user-page.html [hunk 2]
```diff
@@ -55,37 +137,40 @@ <h1 id="page-title">User Page</h1>
...
+	            { model: 'heading3', view: 'h3', title: 'Subheading', class: 'ck-heading_heading3' } 
+	          ]
+                }
+              } )
+              .then( editor => { console.log( editor ); })
+              .catch( error => { console.log( error ); });
+          </script> 
+        </form>
+      </article>
+
```
Comment (FluffySheep) [inline src/main/webapp/user-page.html]: nit: remove extra space

**Comment by FluffySheep** [comment]
Please fix the two spacing issues before merging. Thanks!

**Pull request merged**